eZ/Publish/API/Repository/Tests/TrashServiceTest.php

```diff
@@ -669,7 +669,7 @@ public function testFindTrashItems()
         $query = new Query();
         $query->filter = new Criterion\LogicalAnd(
             [
-                new Criterion\Field('title', Criterion\Operator::LIKE, '*'),
+                new Criterion\ContentTypeId(1),
```

Comment:
There are more Criteria supported by Trash filtering, but the coverage is missing.